Sources/ComposableArchitectureMacros/ReducerMacro.swift

```diff
@@ -361,13 +367,30 @@ extension ReducerMacro: MemberMacro {
         )
       }
       if !hasBody {
+        var staticVarBody = ""
+        if reducerTypeScopes.isEmpty {
+          staticVarBody = "EmptyReducer<Self.State, Self.Action>"
+        } else if reducerTypeScopes.count == 1 {
+          staticVarBody = reducerTypeScopes[0]
+        } else {
+          for _ in 1...(reducerTypeScopes.count - 1) {
+            staticVarBody.append("ReducerBuilder<Self.State, Self.Action>._Sequence<")
+          }
+          staticVarBody.append(reducerTypeScopes[0])
+          staticVarBody.append(", ")
+          for type in reducerTypeScopes.dropFirst() {
+            staticVarBody.append(type)
+            staticVarBody.append(">, ")
+          }
+          staticVarBody.removeLast(2)
+        }
```

Comment:
This is messy code, but it essentially generates a type like this:

```swift
ReducerBuilder<Self.State, Self.Action>._Sequence<
  ReducerBuilder<Self.State, Self.Action>._Sequence<
    ReducerBuilder<Self.State, Self.Action>._Sequence<
      Scope<Self.State, Self.Action, Feature1>, 
      Scope<Self.State, Self.Action, Feature2>
    >, 
    Scope<Self.State, Self.Action, Feature3>
  >, 
  Scope<Self.State, Self.Action, Feature4>
>
```

…from a list of `Scope` types.